docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md

```diff
@@ -0,0 +1,51 @@
+# Add Mattermost notification configuration
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add a specific configuration to get notifications delivered on Mattermost.
+
+## Prerequisites
+
+To add Mattermost notification configurations you need
+
+- A Netdata Cloud account
+- Access to the space as an **administrator**
+- Space needs to be on **Business** plan or higher
+- Have a Mattermost app on your workspace to receive the webhooks, for mode details check [how to configure this on Mattermost](#settings-on-mattermost)
```

Comment:
Small typo `mode` -> `more`

Reply:
(And seems the same typo in other methods too).

Reply:
Nice catch. Copy/paste from the another integration, so once again I will update them all.